nextflow_schema.json

```diff
@@ -13,14 +13,15 @@
             "required": ["input", "outdir"],
             "properties": {
                 "input": {
+                    "description": "Path to comma-separated file containing information about the samples in the experiment.",
+                    "help_text": "A design file with information about the samples in your experiment. Use this parameter to specify the location of the input files. It has to be a comma-separated file with a header row. See [usage docs](https://nf-co.re/sarek/usage#input).\n\nIf no input file is specified, sarek will attempt to locate one in the `{outdir}` directory. If no input should be supplied, i.e. when --step is supplied or --build_from_index, then set --input false",
```

Comment:
Mention's sarek here.
Does it do these things currently?

Reply:
Bad copy paste... let me fix that